config/database.yml

```diff
@@ -1,25 +1,47 @@
-# SQLite version 3.x
-#   gem install sqlite3
+# PostgreSQL. Versions 8.2 and up are supported.
 #
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem 'sqlite3'
+# Install the pg driver:
```

Comment:
Remove ALL the un-necessary comments, always ensure your files are lean and direct, the comments make it hard to skim through ...